packages/avatar/README.md

```diff
@@ -0,0 +1,285 @@
+# Avatars`RMWC ADDON`
+
+> Avatars are virtual representations of users in a system.
+
+-   Module __@rmwc/avatar__
+-   Import styles:
+    -   Using CSS Loader
+        -   import __'@rmwc/avatar/styles';__
+    -   Or include stylesheets
+        -   __'@rmwc/avatar/avatar.css'__;
+        -   __'@rmwc/icon/icon.css'__;
+        -   __'@material/ripple/dist/mdc.ripple.css'__;
+
+Images
+
+```js
+
+<\>
+
+  <Avatar
+
```

Comment:
There is a lot of unnessecary spacing here. But I guess we can't do much about it?

Reply:
Yes, it is annoying with the spaces. I don't think we can fix it with the current markdown lib, but there might be other options. We could also add prettier to the build process or as a commit hock to fix this. This might be a good idea anyway, but might affect many files, so it should be in a separate PR.

Reply:
I have added prettier  to the script, but it doesn't fix all the issues